config/passport-jwt.js

```diff
@@ -0,0 +1,55 @@
+const passport = require("passport");
+const passportJWT = require("passport-jwt");
+const jwt = require("jsonwebtoken");
+require('dotenv').config();
+const User = require("../service/schemas/user");
+
+const JWTStrategy = passportJWT.Strategy;
+const ExtractJWT = passportJWT.ExtractJwt;
+
+const opts = {
+  jwtFromRequest: ExtractJWT.fromAuthHeaderAsBearerToken(),
+  secretOrKey: process.env.JWT_SECRET,
+};
+
+
+passport.use(new JWTStrategy(opts, async (payload, done) => {
+    try {
+      const user = await User.findOnde({id:payload.sub});
+      if (!user) {
+        return done(null, false);
+      }
+      return done(null, user);
+    } catch (error) {
```

Comment:
```suggestion
    } catch (error) {
    console.error(error)
```
To ci powie co sie tu dzieje